android/app/src/main/java/com/mattermost/rnbeta/MainApplication.java

```diff
@@ -55,20 +57,22 @@ public boolean isDebug() {
             new SvgPackage(),
             new LinearGradientPackage(),
             new OrientationPackage(),
-            new RNNotificationsPackage(MainApplication.this)
+            new RNNotificationsPackage(MainApplication.this),
```

Comment:
Why not just pass `this` here? `createAdditionalReactPackages()` isn't a static method, so there should be no need to access this variable statically